pyk/.cruft.json

```diff
@@ -0,0 +1,18 @@
+{
+  "template": "https://github.com/runtimeverification/python-project-template.git",
+  "commit": "46685b928f6e7765d42b98c59cc0cdc5217e12f9",
+  "checkout": null,
+  "context": {
+    "cookiecutter": {
+      "project_name": "pyk",
+      "project_slug": "pyk",
+      "package_name": "pyk",
+      "version": "0.1.0",
```

Comment:
This is pyk version, or template version?

Reply:
If it's pyk version, we should probably try to make sure this stays in sync with actual version (like how KEVM makes sure on CI that they stay in sync)

Reply:
This is the `version` parameter value passed to `cookiecutter` when first generating the project; `cruft` uses this information to regenerate the template for `cruft update`, i.e. when we want to apply a new version of the template to the project. So it is technically `pyk` version, but it is not supposed to be modifed.